src/pages/trading-specification/components/_elements.tsx

```diff
@@ -236,12 +235,22 @@ export const TableCellGroup = ({ data, market }: TTableCellGroup) => {
     const { symbol, instrument: text, dl_icon, swf_icon } = data
     const [show_popUp, setShowPopUp] = useState(false)
     const [popup_type, setPopupType] = useState<TPopupType>()
+    const [scrollY, setScrollY] = useState<number>()
 
     useEffect(() => {
-        document.body.style.overflow = show_popUp ? 'hidden' : 'scroll'
+        document.body.style.position = show_popUp ? 'fixed' : ''
+        document.body.style.width = show_popUp ? '100%' : ''
+        if (!show_popUp && scrollY) {
+            window.scrollTo({ top: scrollY, behavior: 'smooth' })
+        }
+        const styleTag = document.createElement('style')
+        styleTag.type = 'text/css'
+        styleTag.innerHTML = `body { overflow: ${show_popUp ? 'hidden' : 'unset'} !important; }`
+        document.body.insertAdjacentElement('beforeend', styleTag)
```

Comment:
Do we need to insert style tag to control the css rule?  `document.body.style.` can't do the work here?

Reply:
It working earlier with `document.body.style`, But currently when changing the language we have a style make the body `overflow: auto !important`. Which causes background scrollable and `document.body.style` doesn't work.
[https://github.com/binary-com/deriv-com/pull/4912](url)

Reply:
Since we are making behaviour consistent for both mobile and web, this set of code no longer required.